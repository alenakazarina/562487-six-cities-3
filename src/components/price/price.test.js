import React from 'react';
import renderer from 'react-test-renderer';
import Price from './price';

const prices = [200, 300];
const prefixes = [`place-card`, `property`];

describe(`Rating`, () => {
  it(`should render place card's Price`, () => {
    const tree = renderer.create(
        <Price prefix={prefixes[0]} price={prices[0]} />
    ).toJSON();
    expect(tree).toMatchSnapshot();
  });
  it(`should render property's Price`, () => {
    const tree = renderer.create(
        <Price prefix={prefixes[1]} price={prices[1]} />
    ).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
