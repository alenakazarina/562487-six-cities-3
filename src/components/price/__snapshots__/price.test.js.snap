// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Rating should render place card's Price 1`] = `
<div
  className="place-card__price"
>
  <b
    className="place-card__price-value"
  >
    €
    200
  </b>
  <span
    className="place-card__price-text"
    style={
      Object {
        "marginLeft": "3px",
      }
    }
  >
    / night
  </span>
</div>
`;

exports[`Rating should render property's Price 1`] = `
<div
  className="property__price"
>
  <b
    className="property__price-value"
  >
    €
    300
  </b>
  <span
    className="property__price-text"
    style={
      Object {
        "marginLeft": "3px",
      }
    }
  >
     night
  </span>
</div>
`;
